waspc/data/Generator/templates/react-app/src/router.jsx

```diff
@@ -25,18 +25,20 @@ const router = (
     {=^ rootComponent.isDefined =}
     <>
     {=/ rootComponent.isDefined =}
```

Comment:
I don't think we need this anymore - it was only here to ensure `Router` had a single child component. The `Switch` should take care of that from now on.

Reply:
Removed it